src/load/azext_load/data_plane/load_test/params.py

```diff
@@ -52,6 +53,7 @@ def load_arguments(self, _):
         c.argument("autostop", argtypes.autostop)
         c.argument("autostop_error_rate", argtypes.autostop_error_rate)
         c.argument("autostop_error_rate_time_window", argtypes.autostop_error_rate_time_window)
+        c.argument("regionwise_engines", argtypes.regionwise_engines)
```

Comment:
For my understanding 2: In case of both the argument and yaml, arg takes precedence?

Reply:
yup